leg_detector/src/leg_detector.cpp

```diff
@@ -69,7 +69,7 @@ using namespace MatrixWrapper;
 static double no_observation_timeout_s = 0.5;
 static double max_second_leg_age_s     = 2.0;
 static double max_track_jump_m         = 1.0;
-static double max_meas_jump_m          = 0.75; // 1.0
+static double max_meas_jump_m          = 1.0;
 static double leg_pair_separation_m    = 1.0;
 static string fixed_frame              = "odom_combined";
```

Comment:
Changed to "odom".

Reverted the change to max_meas_jump.

This file is anyways not being used for our purposes.